File: IOT/anshulgurjar53_AnshulSinghGurjar_2024it1193_2/FunctionAndArrays/SpanOfArray.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();int a[]=new int[n];
    
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int max=a[0],min=a[0];
    for(int i=0;i<n;i++)
    {
        if(a[i]>max)
          max=a[i];
        else if(a[i]<min)
          min=a[i];
    }
    System.out.println(max-min);
 }

}

File: AR-VR/devAyushDubey_ayushdubey_2024csit1122_2/FunctionAndArrays/SpanOfArray.cpp
#include<iostream>
using namespace std;

int main(){
    int n,i,num;
    cin>>n;
    int max = 0, min;
    for(i=0;i<n;i++){
        cin>>num;
        if(num>max)
            max=num;
        if(i==0)
            min=num;
        else if(num<min)
            min=num;
    }
    cout<<max-min<<endl;
    //write your code here
    
}